### PR TITLE
358, 357, 360 Application improvements and fixed stmas address

### LINKED ISSUE #357: Change address in "About" screen
**Describe the bug**
It is "Winzererstraße 9" instead of "Winzererstraße 7"

### LINKED ISSUE #358: Application View in flutter often crashes
**Describe the bug**
The application view in flutter sometimes shows weird behavior:
Sometimes multiples text inputs have focus and then changing their inputs shows undefinable behavior.

**To Reproduce**
1. Go to tab Ausweisen, click on "Ehrenamtskarte beantragen"
2. Fill in all details (e.g. for blue card, juleica)
3. At some point go through all of the steps above and click on other inputs arbitrarily
4. Go to "Persönliche Daten"
5. Try to edit some text inputs there.

**Screenshots**
image

**Additional context**
Happens both on Android and iOS.
The console shows the following error message:
```
E/flutter ( 3977): [ERROR:flutter/lib/ui/ui_dart_state.cc(186)] Unhandled Exception: A FocusNode was used after being disposed.
E/flutter ( 3977): Once you have called dispose() on a FocusNode, it can no longer be used.
E/flutter ( 3977): #0      ChangeNotifier._debugAssertNotDisposed.<anonymous closure> (package:flutter/src/foundation/change_notifier.dart:117:9)
E/flutter ( 3977): #1      ChangeNotifier._debugAssertNotDisposed (package:flutter/src/foundation/change_notifier.dart:123:6)
E/flutter ( 3977): #2      ChangeNotifier.notifyListeners (package:flutter/src/foundation/change_notifier.dart:234:12)
E/flutter ( 3977): #3      FocusNode._notify (package:flutter/src/widgets/focus_manager.dart:1052:5)
E/flutter ( 3977): #4      FocusManager._applyFocusChange (package:flutter/src/widgets/focus_manager.dart:1771:12)
E/flutter ( 3977): #5      _rootRun (dart:async/zone.dart:1346:47)
E/flutter ( 3977): #6      _CustomZone.run (dart:async/zone.dart:1258:19)
E/flutter ( 3977): #7      _CustomZone.runGuarded (dart:async/zone.dart:1162:7)
E/flutter ( 3977): #8      _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1202:23)
E/flutter ( 3977): #9      _rootRun (dart:async/zone.dart:1354:13)
E/flutter ( 3977): #10     _CustomZone.run (dart:async/zone.dart:1258:19)
E/flutter ( 3977): #11     _CustomZone.runGuarded (dart:async/zone.dart:1162:7)
E/flutter ( 3977): #12     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1202:23)
E/flutter ( 3977): #13     _microtaskLoop (dart:async/schedule_microtask.dart:40:21)
E/flutter ( 3977): #14     _startMicrotaskLoop (dart:async/schedule_microtask.dart:49:5)
```

We are not alone with this. Similar issues exist for flutter_form_builder:
https://github.com/danvick/flutter_form_builder/issues/313
https://github.com/danvick/flutter_form_builder/issues/683

### LINKED ISSUE #360: Attachments of applications are not deleted
Currently, when we delete an application via GraphQl, only the DB entry for that application gets deleted.
The attachments of the application which are stored on disk are not yet deleted.
They should be deleted as well.

### PR DESCRIPTION
**Pull request by RoAnBu**
closes #358 
closes #357 
closes #360

### CODE REVIEW

**Review by Schedulaar — APPROVED** [review]

File: backend/src/main/kotlin/app/ehrenamtskarte/backend/application/database/repos/EakApplicationRepository.kt [hunk 1]
```diff
@@ -75,17 +75,18 @@ object EakApplicationRepository {
...
         }
     }
 
-    fun delete(applicationId: Int): Boolean {
+    fun delete(applicationId: Int, graphQLContext: GraphQLContext): Boolean {
         return transaction {
             val application = EakApplicationEntity.findById(applicationId)
                 if (application != null) {
+                    val applicationDirectory = File(graphQLContext.applicationData, application.id.toString())
                     application.delete()
```
Comment (maxammann) [inline backend/src/main/kotlin/app/ehrenamtskarte/backend/application/database/repos/EakApplicationRepository.kt:88]: Is it possible that this throws an exception and the next line is not executed?

**Review by maxammann — APPROVED** [review]

File: backend/src/main/kotlin/app/ehrenamtskarte/backend/application/database/repos/EakApplicationRepository.kt (hunk 1 above)
Comment (Schedulaar) [reply backend/src/main/kotlin/app/ehrenamtskarte/backend/application/database/repos/EakApplicationRepository.kt:88]: I guess, this is not reaally a problem... If the above throws an exception, the frontend also communicates taht there was a problem deleting the application and hence will still be present in the list of applications.
Then, they'd have to call an administrator anyway why this application cannot be deleted...

**Review by Schedulaar — APPROVED** [review]
@RoAnBu Can we merge this soon?

**Comment by RoAnBu** [comment]
@Schedulaar What do you think about Max' comment? We probably need some error management there?

**Comment by Schedulaar** [comment]
As I wrote in a response to max' comment, I don't think it's a huge problem we need to consider now.

**Pull request merged**